parse/DoubleValueRefParser.cpp

```diff
@@ -116,6 +117,10 @@ parse::double_parser_rules::double_parser_rules(
         =   int_rules.statistic_expr [ _val = construct_movable_(new_<ValueRef::StaticCast<int, double>>(deconstruct_movable_(_1, _pass))) ]
         ;
 
+    int_total_fighter_shots_cast
```

Comment:
is an int -> double cast not automatic?

Reply:
not sure if/how automatic cast is supposed to work - but it sure does not work without that extra cases

Reply:
OK